media/css/mozorg/home/home.scss

```diff
@@ -472,7 +472,7 @@ main {
         padding-top: 40px;
 
         &:before {
-            @include at2x('/media/img/firefox/common/firefox-logo.png');
+            @include at2x('/media/img/firefox/template/logo.png');
```

Comment:
I think this should use `logo-large.png` too, else it looks fuzzy on high-res screens here.